lib/app/lift.js

```diff
@@ -65,11 +66,11 @@ function _printSuccessMsg(sails) {
 
     sails.log.ship && sails.log.ship();
     sails.log.info(('Server lifted in `' + sails.config.appPath + '`'));
-    sails.log.info(('To see your app, visit ' + (sails._getBaseUrl() || '').underline));
+    sails.log.info(chalk.underline('To see your app, visit ' + (sails._getBaseUrl() || '')));
     sails.log.info(('To shut down Sails, press <CTRL> + C at any time.'));
     sails.log.blank();
-    sails.log('--------------------------------------------------------'.grey);
-    sails.log((':: ' + new Date()).grey);
+    sails.log();
```

Comment:
just a heads up-- I believe this changes the logs.  (Probably fine, but just wanted to point it out)